kinode/packages/kns_indexer/kns_indexer/src/lib.rs

```diff
@@ -112,16 +111,22 @@ fn subscribe_to_logs(eth_provider: &eth::Provider, from_block: u64, filter: eth:
             }
         }
     }
-    #[cfg(not(feature = "simulation-mode"))]
     println!("subscribed to logs successfully");
 }
 
 call_init!(init);
 fn init(our: Address) {
+    #[cfg(feature = "simulation-mode")]
+    let (chain_id, contract_address) = (31337, KNS_LOCAL_ADDRESS.to_string());
+    #[cfg(not(feature = "simulation-mode"))]
     let (chain_id, contract_address) = (10, KNS_OPTIMISM_ADDRESS.to_string());
-
     println!("indexing on contract address {}", contract_address);
 
+    #[cfg(feature = "simulation-mode")]
+    let kns_first_block: u64 = 1;
+    #[cfg(not(feature = "simulation-mode"))]
+    let kns_first_block: u64 = 114_923_786;
```

Comment:
these can and should still be consts behind a feature flag

Reply:
these are consts now, added ones for app_store too.